org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/launch/GradleRunConfigurationAttributes.java

```diff
@@ -258,88 +290,83 @@ public static void applyShowConsoleView(boolean showConsoleView, ILaunchConfigur
         launchConfiguration.setAttribute(SHOW_CONSOLE_VIEW, showConsoleView);
     }
 
+    public static void applyWorkspaceOverride(boolean overrideWorkspaceSettings, boolean offlineMode, boolean buildScansEnabled, ILaunchConfigurationWorkingCopy launchConfiguration) {
```

Comment:
Hmmm can I click `apply` on individual pages? If not then the model doesn't match the reality of the UI. Nothing we need to fix right now, but something worth putting in the backlog for cleanup.